include/cudaq/Support/Placement.h

```diff
@@ -0,0 +1,64 @@
+/****************************************************************-*- C++ -*-****
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#pragma once
+
+#include "cudaq/Support/Device.h"
+
+namespace cudaq {
+
+class Placement {
```

Comment:
Document the class `Placement`? Indeed, none of the methods, etc. are documented here.

Reply:
Updated in 9f60585e